frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PdfConfig/PdfConfig.tsx

```diff
@@ -1,33 +1,25 @@
 import React, { useRef, useState } from 'react';
-import { StudioProperty } from '@studio/components';
-import { useTranslation } from 'react-i18next';
-import { FileIcon } from '@studio/icons';
 import { usePdf } from '../../../../hooks/usePdf/usePdf';
 import { ConvertChoicesModal } from './ConvertPageToPdfWhenExistingModal/ConvertChoicesModal';
 import { useSavableFormLayoutSettings } from '@altinn/ux-editor/hooks/useSavableFormLayoutSettings';
+import { PdfConfigCard } from './PdfConfigCard';
 
 export const PdfConfig = () => {
-  const { t } = useTranslation();
-  const { isCurrentPagePdf, getPdfLayoutName, convertCurrentPageToPdf, convertExistingPdfToPage } =
-    usePdf();
+  const { getPdfLayoutName, convertCurrentPageToPdf } = usePdf();
   const [isPdfUpdated, setIsPdfUpdated] = useState(false);
   const savableLayoutSettings = useSavableFormLayoutSettings();
   const convertChoicesDialogRef = useRef<HTMLDialogElement>(null);
 
   const handleClickConvertButton = () => {
     if (!!getPdfLayoutName()) {
```

Comment:
This is strictly speaking not part of your PR, but i suggest refactoring this while you're at it. I think this clarifies the logic a bit, but on the other side it might be too verbose. What do you think? Either way i think we generally try to use `Boolean()` instead of `!!`

```suggestion
  const shouldShowModal = () => {
  	return Boolean(getPdfLayoutName())
  }

  const handleClickConvertButton = () => {
    if (shouldShowModal()) {
```

Reply:
Totally agree! 😄 
However, I've changed it to this instead to simplify it even further:
```tsx
const shouldShowModal = Boolean(getPdfLayoutName()); 
```

Reply:
Even better! Nice!